modules/dmrpp_module/build_dmrpp.cc

```diff
@@ -834,7 +898,7 @@ int main(int argc, char *argv[]) {
         // given HDF5 dataset
         if (!dmr_name.empty()) {
             // Get dmr:
-            unique_ptr<DMRpp> dmrpp(new DMRpp);
+            shared_ptr<DMRpp> dmrpp(new DMRpp);
```

Comment:
shared_ptr<> works here, but it is less efficient than unique_ptr<>. In some of the profiling, I saw shared_ptr<> using much more time that I originally expected.